### DISCUSSION TITLE
blog/rime-configuration-manager/

### DISCUSSION THREAD

**Discussion by giscus** [rime/home#1329] [General]
# blog/rime-configuration-manager/

Le blog de la Rime

https://rime.im/blog/rime-configuration-manager/

**Comment by CN-make** [comment]
所以說目前是停止維護的一個狀態嗎？

**Reply by MokOopsing** [reply]
[東風破 /plum/](https://github.com/rime/plum) 本身没有什么可更新的，目前是可用状态

**Reply by fxliang** [reply]
可能东风破要加一个从注册表读取用户目录的功能，早前有人提安装路径不对的问题
